gaptrain/active.py

```diff
@@ -171,13 +280,16 @@ def get_active_configs(config, gap, method_name,
 
     :param e_thresh: (float) Energy threshold in eV above which the MD frame
                      is returned by the active learning function i.e
-                     E_t < |E_GAP - E_true|
+                     E_t < |E_GAP - E_true|  method=='true'
```

Comment:
This confused me as I thought it was something to do with the True boolean

Reply:
good point